source/includes/fact-atlas-compatible.rst

```diff
@@ -0,0 +1,4 @@
+You can connect using the Node.js driver and |page-topic| for
```

Comment:
I: To avoid "using" and also make this a little clearer, I'd go with something like this:

```suggestion
You can use the {+driver-short+} to connect and |page-topic| for
```